providers-sdk/v1/resources/resources.proto

```diff
@@ -40,6 +40,7 @@ message ResourceInfo {
   string min_mondoo_version = 25;
   string defaults = 26;
   string provider = 27;
+  repeated ResourceInfo others = 29;
```

Comment:
nit: would you mind adding a comment above this and the `others` field below:

```
// This field contains references to other providers with the same resource/field.
// Note: Please do not use this field, it is only temporary and will be removed
// in the future once binding resources are mandatory for all executions.
```

(if this merges first no worries, we can add it later)